### DISCUSSION TITLE
[Project Management] Weekly meeting and other project management processes

### DISCUSSION THREAD

**Discussion by ChristinaLK** [PelicanPlatform/pelican#1141] [Ideas]
Christina has made a document to be the agenda/notes space for the Pelican weekly meeting. 

Should develop a plan for how it is populated, who facilitates, who takes notes, who sends a follow up email (if desired). 

Meta-meta issue: is Github the right place to do project management action items like this?

**Comment by turetske** [comment]
@bbockelm Pinging you about the meta-meta issue
